Assets/GameAssets/Boss/BossScripts/Attacks/Lockdown/LockdownAttackEnemyController.cs

```diff
@@ -120,8 +120,8 @@ private void SpawnCoreEnemy()
         //Spawns the core
         _instantiatedCoreEnemy = 
             Instantiate(_coreEnemyPrefab, _patrolLocation.CoreSpawnPoint.position, Quaternion.identity,transform);
-        AmbienceManager.APlayAmbienceOnObject?.Invoke(FmodAmbienceEvents.Instance.LimbIdle, _instantiatedCoreEnemy);
-        RuntimeSfxManager.APlayOneShotSfx?.Invoke(FmodSfxEvents.Instance.LimbSpawn, _patrolLocation.CoreSpawnPoint.position);
+        /*AmbienceManager.APlayAmbienceOnObject?.Invoke(FmodAmbienceEvents.Instance.LimbIdle, _instantiatedCoreEnemy);
+        RuntimeSfxManager.APlayOneShotSfx?.Invoke(FmodSfxEvents.Instance.LimbSpawn, _patrolLocation.CoreSpawnPoint.position);*/
```

Comment:
This probably shouldn't be commented out. I know there was an error relating to this code but I believe Andrea already fixed it